charts/seed-monitoring/charts/core/charts/prometheus/values.yaml

```
@@ -162,17 +174,53 @@ allowedMetrics:
   - process_max_fds
   - process_open_fds
   machineControllerManager:
-  - process_max_fds
-  - process_open_fds
+  - mcm_cloud_api_requests_failed_total
+  - mcm_cloud_api_requests_total
   - mcm_machine_controller_frozen
+  - mcm_machine_current_status_phase
+  - mcm_machine_deployment_failed_machines
   - mcm_machine_items_total
-  - mcm_machineset_items_total
   - mcm_machine_set_failed_machines
   - mcm_machinedeployment_items_total
-  - mcm_machine_deployment_failed_machines
-  - mcm_cloud_api_requests_total
-  - mcm_cloud_api_requests_failed_total
+  - mcm_machineset_items_total
   - mcm_scrape_failure_total
+  - machine_adds
+  - machine_depth
+  - machine_queue_latency
+  - machine_retries
+  - machine_work_duration
+  - machinedeployment_adds
+  - machinedeployment_depth
+  - machinedeployment_queue_latency
+  - machinedeployment_retries
+  - machinedeployment_work_duration
+  - machinesafetyapiserver_adds
+  - machinesafetyapiserver_depth
+  - machinesafetyapiserver_queue_latency
+  - machinesafetyapiserver_retries
+  - machinesafetyapiserver_work_duration
+  - machinesafetyorphanvms_adds
+  - machinesafetyorphanvms_depth
+  - machinesafetyorphanvms_queue_latency
+  - machinesafetyorphanvms_retries
+  - machinesafetyorphanvms_work_duration
+  - machinesafetyovershooting_adds
+  - machinesafetyovershooting_depth
+  - machinesafetyovershooting_latency
+  - machinesafetyovershooting_retries
+  - machinesafetyovershooting_work_duration
+  - machineset_adds
+  - machineset_depth
+  - machineset_queue_latency
+  - machineset_retries
+  - machineset_work_duration
+  - node_adds
+  - node_depth
+  - node_queue_latency
+  - node_retries
+  - node_work_duration
```

```suggestion
  - node_work_duration
  - secret_adds
  - secret_depth
  - secret_queue_latency
  - secret_retries
  - secret_work_duration
```
We could track the metrics for secrets as well?

---

SureI  will add `secrets` as well. What about the `aws`, `azure`, `gcp`, `openstack`, `packet` machineclass controller metrics? Should we also display them?

---

I thought about it too. But we soon going to release an MCM with a generic machine class instead of all of these. Maybe we can integrate the metrics for the generic machine class then? Or we could introduce for all specific machine classes now, and update it to generic later. Either way sounds good to me. I will leave it to you.

---

And adding secrets should help. Thanks.

---

I have added the metrics for the `secret` control loop. Let's wait for the generic machine class controller metrics.